dev/io.openliberty.microprofile.metrics.internal.3.0.monitor/src/io/openliberty/microprofile/metrics/internal/monitor/MetricsJaxRsEMCallbackImpl.java

```diff
@@ -37,9 +41,9 @@
 				"service.vendor=IBM" })
 public class MetricsJaxRsEMCallbackImpl  implements DefaultExceptionMapperCallback {
 
-	
+    private static final TraceComponent tc = Tr.register(MetricsJaxRsEMCallbackImpl.class);
```

Comment:
Does this get used anywhere?

Reply:
Ah nope. That was left over when it originally was printing an warning.